src/System.Threading.Tasks.Extensions/tests/AsyncValueTaskMethodBuilderTests.cs

```diff
@@ -0,0 +1,131 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace System.Threading.Tasks.Tests
+{
+    public class AsyncValueTaskMethodBuilderTests
+    {
+        [Fact]
+        public void Create_ReturnsDefaultInstance()
+        {
+            AsyncValueTaskMethodBuilder<int> b = ValueTask<int>.CreateAsyncMethodBuilder();
+            Assert.Equal(default(AsyncValueTaskMethodBuilder<int>), b);
```

Comment:
This was a bit confusing to me at first, because `AsyncValueTaskMethodBuilder.Create` does assign to its `_methodBuilder` field. But it turns out that it assigns it to `default` anyways (or technically calls a  method returning `default` and assigns that). I guess we would need to change this test if that ended up returning something else? (There's a comment suggesting we might [here](https://github.com/dotnet/coreclr/blob/ce3ff76234bf199b4498a5d31f05af6eb43073f6/src/mscorlib/src/System/Runtime/CompilerServices/AsyncMethodBuilder.cs#L290))